packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -27,12 +27,12 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone {
     /**
      * @notice The current Semver of the FaultDisputeGame implementation.
      */
-    string internal constant VERSION = "0.0.1";
+    string internal constant VERSION = "0.0.2";
 
     /**
      * @notice The max depth of the game.
      */
-    uint256 internal constant MAX_GAME_DEPTH = 63;
+    uint256 internal constant MAX_GAME_DEPTH = 4;
```

Comment:
Could we make this `immutable` rather than a `constant`? This way it can be set correctly in various contexts including tests and in prod.